input/fsh/omrs-related-person.fsh

```diff
@@ -0,0 +1,137 @@
+Profile: OMRSRelatedPerson
+Parent: RelatedPerson
+Id: omrs-related-person
+Title: "OpenMRS Related Person"
+Description: "A FHIR RelatedPerson as understood by OpenMRS Relationship"
+* id 1..1
+* patient 1..1
+* identifier 1..1
```

Comment:
The cardinality has to be one-to-many for example

Reply:
Updated